.github/workflows/ci.yml

```diff
@@ -17,20 +17,36 @@ jobs:
     runs-on: ubuntu-latest
     env:
       CRYOSTAT_GRAFANA_IMG: quay.io/cryostat/cryostat-grafana-dashboard
+      REF_NAME: ${{ github.ref_name }}
     steps:
     - uses: actions/checkout@v2
     - name: Install qemu
       continue-on-error: false
       run: |
         sudo apt-get update
         sudo apt-get install -y qemu-user-static
+    - name: Check release branch
+      id: check-branch
+      run: |
+        if [[ ${REF_NAME} == cryostat-v* ]]; then
+          echo "image-tag=${REF_NAME:10}-dev" >> $GITHUB_OUTPUT
+        else
+          echo "image-tag=${REF_NAME}" >> $GITHUB_OUTPUT
+        fi
+    - name: Check commit git tag
+      id: check-commit
+      run: |
+        output=$(git describe --tags --exact-match 2>/dev/null || echo -n '')
```

Comment:
So far we have been using git tags like `v2.3.1`, but that should ideally produce image tags `2.3.1`, `2.3`, and `2`. I think just a simple `| sed -e "s/^v//"` should do the trick to strip off the leading `v` if there is one.

```bash
$ echo "v2.3.1" | sed -e "s/^v//"
2.3.1
$ echo "2.3.1" | sed -e "s/^v//"
2.3.1
```